package.json

```diff
@@ -54,12 +54,13 @@
     "element-class": "^0.1.1",
     "extend": "^1.3.0",
     "filter-object": "^1.1.2",
-    "flatsheet-api-client": "^1.2.0",
+    "flatsheet-api-client": "^2.0.0",
```

Comment:
I updated this to reflect the new 'authorization' changes. Before, the authorization header was always included, which was sometimes equal to `:` when no username/password was present. 

This brought a new problem, however, because the call to `flatsheet.sheet` in `browser/sheet.js` is now an `undefined is not a function` error. It looks like the routes are incomplete? Let me know and feel free to revert this change if the 2.x version is not ready.